### PR TITLE
Respect both our pokestop timeout *and* the server timeout.

### PR DESCRIPTION
**Pull request by apottere**
If we haven't visited a pokestop yet, assume the server knows the timer.

### CODE REVIEW

File: src/main/kotlin/ink/abb/pogo/scraper/util/map/Pokestop.kt [hunk 1]
```diff
@@ -11,6 +11,7 @@ package ink.abb.pogo.scraper.util.map
 import com.pokegoapi.api.map.fort.Pokestop
 
 fun Pokestop.canLoot(ignoreDistance: Boolean = false, lootTimeouts: Map<String, Long>): Boolean {
-    val canLoot = lootTimeouts.getOrElse(id, { 0 }) < System.currentTimeMillis()
+    val now = System.currentTimeMillis()
+    val canLoot = lootTimeouts.getOrElse(id, { 0 }) < now && cooldownCompleteTimestampMs < now
```
Comment (jabbink) [inline src/main/kotlin/ink/abb/pogo/scraper/util/map/Pokestop.kt]: Wouldn't `val canLoot = lootTimeouts.getOrElse(id, { cooldownCompleteTimestampMs }) < now` work as well? As far as I know this `Map` is not filled with 0's or values it knows are incorrect.

Comment (apottere) [inline src/main/kotlin/ink/abb/pogo/scraper/util/map/Pokestop.kt]: Yup, I wrote this before the comment on the other PR - I didn't realize it stored blacklist timers for a successful spin.

**Comment by apottere** [comment]
Feel free to squash when you merge.

**Comment by apottere** [comment]
Also, travis is really hurting.

**Pull request merged**